examples/downsample-register.mjs

```diff
@@ -0,0 +1,5 @@
+import { readImageNode } from "@itk-wasm/image-io";
+import { downwsampleNode } from "@itk-wasm/downsample";
+import { elastixNode } from "@itk-wasm/elastix";
+
+const fixedImageFile = 
```

Comment:
Just for my understanding (as I don't know MJS) what does it mean to do `const fixedImageFile =` without an expression at the right-hand-side?